Pod/Classes/APMAlertController.swift

```diff
@@ -3,14 +3,15 @@
 //
 
 import UIKit
-import SnapKit
 
-@objc public enum APMAlertControllerStyle: Int {
+@objc
+public enum APMAlertControllerStyle: Int {
     case alert
     case actionSheet
 }
 
-@objc open class APMAlertController: UIViewController {
+@objc
+open class APMAlertController: UIViewController {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/7b25c9b676cbcd14f599e9b699c32cd1617e4f78/Pod/Classes/APMAlertController.swift#L14"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 232 lines</td>
    </tr>
  </tbody>
</table>